deploy/helm/magda/charts/content-api/templates/init-scss-compiler.yaml

```diff
@@ -16,9 +16,11 @@ spec:
       containers:
       - name: init-scss-compiler
         image: "{{ .Values.image.repository | default .Values.global.image.repository }}/magda-scss-compiler:{{ .Values.image.tag | default .Values.global.image.tag }}"
-        imagePullPolicy: {{ .Values.image.pullPolicy | default .Values.global.image.pullPolicy }}
-        command:
-            - "/usr/src/app/component/setup.sh"
+        imagePullPolicy: Always
```

Comment:
I think this should stay as it is - with the config that was here before it'll be "always" in dev/preview but "ifnotpresent" in prod, which is better.

Wasn't it the job that had the problem with the pull policy?

Reply:
You're right~ 👍 I will change it back~